### ISSUE TITLE
Strand-specific methylation-calling

### ISSUE THREAD

**Issue by AnjuniPeiris** [jts/nanopolish#428]
Hi,
Just wanted to confirm as I couldn't find it in the documentation, does Nanopolish detect methylation on only one strand or both strands? If it's only one strand, is it just the plus strand?
Thanks!

**Comment by AnjuniPeiris** [comment]
Oh ok, thanks! I was just confused because the methylation frequency tsv file doesn't mention the strand.

I raised this issue because I was comparing methylation detection between Tombo and Nanopolish and I got a 44% similarity in sites between the two. I thought this may be because Tombo detects sites on both both the plus and minus strands, and Nanopolish might not. The Tombo output files give a methylation probability for every cytosine in the genome, so I compared these to Nanopolish to find out whether Nanopolish had overlapping sites with the minus strand in Tombo. I used BEDTools to intersect the Tombo plus strand methylation site file with the Nanopolish file, and got sites that overlapped. However, intersecting the Tombo minus strand file with the Nanopolish file gave no overlapping sites, so I thought Nanopolish wasn't detecting methylation on the minus strand.

Is there a way to get Nanopolish to output the methylation sites on minus strand separately from the sites on the plus strand? Sorry, I haven't been able to find this option for methylation-detection in Nanopolish.
